include/integratorxx/quadratures/gausschebyshev1.hpp

```diff
@@ -39,8 +39,7 @@ class GaussChebyshev1
   using point_container = typename base_type::point_container;
   using weight_container = typename base_type::weight_container;
 
-  GaussChebyshev1(size_t npts, point_type lo, point_type up)
-      : base_type(npts, lo, up) {}
+  GaussChebyshev1(size_t npts) : base_type(npts) {}
```

Comment:
This change should really be in a separate commit, no?

Reply:
Do you mean in a separate PR? The changes are all localized to a single [commit](https://github.com/wavefunction91/IntegratorXX/pull/27/commits/44e6c97c3ab24a169b534408643f68cf45a6c0f2) which is documented in the relevant [issue](https://github.com/wavefunction91/IntegratorXX/issues/42#issue-1795532027), so if the change needs to get reverted, it's trackable. If for some reason this PR doesn't get merged (which is not likely), it can always be cherry-picked as a separate PR.

Since there are quite a few commits that depend on it and I'm not a fan of rewriting git history, I'm going to leave it as is. In general I agree with the sentiment of this concern though, will try to be more disciplined on this in the future.

Reply:
Yes, separate PR, I was probably tired :stuck_out_tongue: 

I need this change for the autogenerated Chebyshev quadrature tests...